discussions-api/src/main/java/org/xwiki/contrib/discussions/domain/references/AbstractDiscussionReference.java

```diff
@@ -34,7 +36,7 @@
  * @since 2.0
  */
 @Unstable
-public abstract class AbstractDiscussionReference
+public abstract class AbstractDiscussionReference implements Serializable
```

Comment:
Do we have a rule regarding Serializable and serialVersionUID?

Reply:
I don't recall any